tutorial/consuming_packages/getting_started/build_simple_cmake_project.rst

```diff
@@ -165,13 +165,23 @@ configuration:
    *cmake-build-release*. To do that, just run:
 
 .. code-block:: bash
+    :caption: Windows
+
+    $ conan install . --output-folder build --build=missing
```

Comment:
```suggestion
    $ conan install . --output-folder=build --build=missing
```

Lets be homegeneous in using args =, if ``--build=missing``, better ``--output-folder=build``.

In general lets use ``=`` whenever possible and makes sense, it makes things more explicit and clear